packages/neos-ui/src/Containers/Modals/SelectNodeType/style.css

```diff
@@ -28,3 +28,18 @@
 .gridItem {
     padding: 0;
 }
+
+.nodeTypeDialogHeader {
+    display: flex;
+    justify-content: space-between;
+}
+
+.nodeTypeDialogHeader__filter {
+    margin-right: 40px;
```

Comment:
Don't we have spacing vars for this?

Reply:
Ah, not sure. Could you point me to the place where all vars are set? Did'n found that earlier

Reply:
https://github.com/neos/neos-ui/blob/master/packages/build-essentials/src/webpack.config.js#L59